src/invidious/views/embed.ecr

```diff
@@ -31,7 +31,15 @@
 %>
 </script>
 
-<%= rendered "components/player" %>
+<% if video.reason.nil? %>
+<h3>
+    <%= video.reason %>
+</h3>
+<% else %>
+<div id="player-container" class="h-box">
```

Comment:
I'd prefer to keep a placeholder for the player and put the error message in the middle of that black square (i.e a `p` inside that `#player-container`), like what Youtube does. What do you think?